package.json

```diff
@@ -64,14 +63,15 @@
     "karma-junit-reporter": "1.2.0",
     "karma-spec-reporter": "0.0.32",
     "ng-mocks": "7.7.0",
+    "node-sass": "^4.12.0",
     "prettier": "1.17.0",
     "resize-observer-polyfill": "1.5.1",
     "sass-extract": "2.1.0",
     "sass-extract-loader": "1.1.0",
     "ts-node": "5.0.1",
     "tslint": "5.9.1",
     "tslint-plugin-prettier": "2.0.1",
-    "typescript": "3.1.1",
+    "typescript": "3.5.3",
     "webpack-cli": "^3.3.8"
```

Comment:
We should add `raw-loader` as a dev dependency as well:
`npm install raw-loader -D`